e2e/app/page/conceptset-save-modal.ts

```diff
@@ -52,9 +51,6 @@ export default class ConceptsetSaveModal extends Modal {
     // Click SAVE button.
     await this.clickButton(LinkText.Save, {waitForClose: true});
     await waitWhileLoading(this.page);
-
-    const conceptActionPage = new ConceptsetActionsPage(this.page);
-    await conceptActionPage.waitForLoad();
```

Comment:
ConceptsetSearchPage -> ConceptsetSaveModal -> ConceptsetActionsPage -> ConceptsetSearchPage

Reply:
It would be nice to have a diagram  in  LucidChart or Graphviz to show the structure of your DAG (and thus the navigation structure of the app as a whole).